helm/aws-operator/templates/08-deployment.yaml

```
@@ -20,13 +20,13 @@ spec:
       volumes:
       - name: aws-operator-configmap
         configMap:
-          name: aws-operator-configmap
+          name: aws-operator-configmap.v{{ .Values.version }}
```

What I though may make sense is this. 

```suggestion
          name: configmap-{{ .Values.versionedName }}
```

So we have this in the values file. 

```
aws-operator-.v[[ .Version ]]
```

Otherwise we have this stupid version thing everywhere. But as I said I am not too big on it.

---

I think I know what you mean.